src/babel-serializer/network/pcap.hh

```diff
@@ -55,4 +56,7 @@ struct data
 
 /// reads a pcap dump from memory
 data read(cc::span<std::byte const> data, read_config const& cfg = {}, error_handler on_error = default_error_handler);
+
+/// reads a pcap dump from memory
```

Comment:
"... calling on_header once for the header and then on_packet for each contained packet"

Reply:
Expanded the comment accordingly :+1: